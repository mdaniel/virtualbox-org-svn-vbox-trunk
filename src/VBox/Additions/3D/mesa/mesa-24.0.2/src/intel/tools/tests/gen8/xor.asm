xor(16)         g10<1>UD        g1<0,1,0>UD     g1.1<0,1,0>UD   { align1 1H };
xor(8)          g4<1>UD         g5.6<0,1,0>UD   ~g5.7<0,1,0>D   { align1 1Q };
