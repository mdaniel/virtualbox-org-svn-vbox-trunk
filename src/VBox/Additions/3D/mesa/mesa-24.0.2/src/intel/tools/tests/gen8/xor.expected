07 00 80 00 08 02 40 21 20 00 00 02 24 00 00 00
07 00 60 00 08 02 80 20 b8 00 00 0a bc 40 00 00
